packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -6,12 +6,28 @@ import { TAG_NAMES, INTERNAL_TAG_NAMES } from '@porsche-design-system/shared';
 import type { TagName } from '@porsche-design-system/shared';
 
 const glue = '\n\n';
+// can't resolve @porsche-design-system/components without building it first, therefore we use relative path
+const sourceDirectory = path.resolve('../components/src/components');
+const componentFiles = globby.sync(`${sourceDirectory}/**/*.tsx`);
+
+const getExportFilePath = (source: string, constName: string | string[], tagName: TagName): string => {
+  const [, importPath] = source.match(new RegExp(`${constName}[\\s\\S]+?from '(.+)';`));
```

Comment:
A `string[]` parameter most likely messes up this regex. 
<img width="127" alt="Bildschirm­foto 2023-03-16 um 09 49 59" src="https://user-images.githubusercontent.com/4380295/225563755-eac8a8cd-a708-495e-9e7c-e6927293e1ff.png">